README.md

```diff
@@ -97,7 +97,7 @@ bugsnag {
 If you disable automatic uploading, you can still run the upload task manually, with the uploadBugsnag*Variant*Mapping task:
 
 ```shell
-./gradlew clean assembleRelease packageRelease uploadBugsnagReleaseMapping
+./gradlew clean assembleRelease packageRelease ${moduleName}:uploadBugsnag${variantName}ReleaseMapping
```

Comment:
What's an example of the `moduleName` and `variantName` values? I have a test project but `uploadBugsnagReleaseMapping` works as expected.